core/src/de/adesso/brainysnake/BrainySnake.java

```diff
@@ -47,6 +54,12 @@ public void create() {
 
         texture = new Texture(pixmap);
 
+        try {
+            backgroundSound = Gdx.audio.newMusic(Gdx.files.internal("./core/src/de/adesso/brainysnake/sound/track.mp3"));
+        } catch (Exception e){
```

Comment:
Pls only check specific Exceptions. If this loc would throw an Exception the Stacktrace would be logged anyway so the catch can be deleted.

Reply:
Deleted the try/catch block and the logger.